code/modules/mob/living/carbon/human/human_say.dm

```diff
@@ -69,13 +69,14 @@
 		return FALSE
 	if((breathes && !L) || breathes && L && (L.status & ORGAN_DEAD))
 		return FALSE
-	if(getOxyLoss() > 10 || AmountLoseBreath() >= 8 SECONDS)
-		emote("gasp")
-		return FALSE
 	if(mind)
 		return !mind.miming
 	return TRUE
 
+/mob/living/carbon/human/cannot_speak_loudly()
+	if(getOxyLoss() > 10 || AmountLoseBreath() >= 8 SECONDS)
+		return TRUE
```

Comment:
```suggestion
		return getOxyLoss() > 10 || AmountLoseBreath() >= 8 SECONDS
```

Reply:
you version doesn't even return `FALSE` as boolean. It returns `null`, which actually resolved to `FALSE` but those values are different.

Reply:
And why do you think, that this is `more readable` ?

Reply:
This does need Gaxeer's code.
To make yours work, you'd need to make this an `if else` or something like it, and that'd just be unnecessary when you can do it like this